File: Selenium/Project/WebsiteTitle.py
import unittest
from selenium import webdriver
from selenium.webdriver.common.keys import Keys

class Task1VerifyWebsiteTitle(unittest.TestCase):
    def setUp(self):
        # Set up the Firefox driver
        self.driver = webdriver.Firefox()

    def test_verify_website_title(self):
        # Navigate to the website
        self.driver.get("https://alchemy.hguy.co/jobs")

        # Get the title of the website
        website_title = self.driver.title

        # Verify the title matches the expected title
        expected_title = "Alchemy Jobs – Job Board Application"
        self.assertEqual(website_title, expected_title, f"Expected title: {expected_title}, Actual title: {website_title}")

    def tearDown(self):
        # Close the browser
        self.driver.close()

if __name__ == "__main__":
    unittest.main()

File: Selenium/Project/WebsiteTitle.java
package Project;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class WebsiteTitle {
        private WebDriver driver;
        private String baseUrl = "https://alchemy.hguy.co/jobs/";

        @BeforeClass
        public void setUp() {
            System.setProperty("webdriver.gecko.driver", "C://Users//001ZXG744//IdeaProjects//geckodriver.exe");
            driver = new FirefoxDriver();
            driver.get(baseUrl);
        }

        @Test
        public void testWebsiteTitle() {
            // Get the title of the website
            String actualTitle = driver.getTitle();

            // Verify the title matches the expected title
            String expectedTitle = "Alchemy Jobs – Job Board Application";
            Assert.assertEquals(actualTitle, expectedTitle, "Website title does not match the expected title.");
        }

        @AfterClass
        public void tearDown() {
            driver.quit();
        }
    }
